web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@momentum-ui/web-components",
-  "version": "2.1.9",
+  "version": "2.1.8",
```

Comment:
version doesn't meet right.

Reply:
We should discuss. 2.1.8 is what's currently published, so I re-wound it here so that when I do an `npm version patch` it will get up to 2.1.9 with a merge.

Reply:
My bad, got it.